tx/textual/valuerenderer/valuerenderer.go

```diff
@@ -61,6 +61,10 @@ func (r Textual) GetValueRenderer(fd protoreflect.FieldDescriptor) (ValueRendere
 				return nil, fmt.Errorf("got empty value renderer for scalar %s", scalar)
 			}
 
+			if scalar == "cosmos.Int" {
+				return NewIntValueRenderer(fd), nil
+			}
+
```

Comment:
@amaurym added this handle few tests, any better way?

Reply:
No, we shouldn't hardcode "cosmos.Int", it's the same issue as https://github.com/cosmos/cosmos-sdk/pull/13696#discussion_r1021520468

Here's a way I'm proposing: https://github.com/cosmos/cosmos-sdk/compare/ap/parse-coins...am/parse-coins?expand=1. Let me know what you think.

Reply:
awesome, looking perfect thanks!